tests/testTriggers.go

```diff
@@ -4548,6 +4552,92 @@ func TriggerPoolAddDisk(contexts *[]*scheduler.Context, recordChan *chan *EventR
 	updateMetrics(*event)
 }
 
+// TriggerPoolAutoDisk performs auto-disk on the storage pools for the given contexts
+func TriggerPoolAutoDisk(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer ginkgo.GinkgoRecover()
+	defer endLongevityTest()
+	startLongevityTest(PoolAutoDisk)
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: PoolAutoDisk,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+	setMetrics(*event)
+	chaosLevel := getPoolExpandPercentage(PoolAutoDisk)
+	stepLog := fmt.Sprintf("get storage pools and perform auto-disk by %v percentage on it ", chaosLevel)
```

Comment:
please change "auto-disk" to "auto" , as we have only "add-disk", "resize-disk", "auto"

Reply:
Done. Changed auto-disk to auto everywhere.

Reply:
Added Expansion prefix.